base/docs/basedocs.jl

```diff
@@ -602,6 +602,25 @@ the last expression in the function body.
 """
 kw"function"
 
+"""
+    ->
+
+`->` is used to define anonymous functions:
```

Comment:
Maybe something like this?
```suggestion
    x -> y

Create an anonymous function mapping argument(s) `x` to the function body `y`.
```